contrib/camel/pkg/apis/sources/v1alpha1/camelsource_types.go

```diff
@@ -98,11 +98,7 @@ type CamelSourceOriginComponentSpec struct {
 
 // CamelSourceStatus defines the observed state of CamelSource
 type CamelSourceStatus struct {
-	// Conditions holds the state of a source at a point in time.
-	// +optional
-	// +patchMergeKey=type
-	// +patchStrategy=merge
-	Conditions duckv1alpha1.Conditions `json:"conditions,omitempty" patchStrategy:"merge" patchMergeKey:"type"`
+	duckv1alpha1.Status `json:",inline"`
```

Comment:
I don't think this comment will stay up to date for long. I am pushing back.

Reply:
That gives me pause.

This embed is published API that users and integrators see and work with and build knowledge and products around. If we rename `ObservedGeneration` to `SeenGeneration`, that's a breaking API change, and it should be documented.

If `duckv1alpha1.Status` is going to be a shared interface, it can't change very often, and when it does, that change should be announced in every project that uses it.